bundles/SeoBundle/src/Controller/Document/DocumentController.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\SeoBundle\Controller\Document;
+
+
+
+use Pimcore\Bundle\AdminBundle\Controller\Admin\ElementControllerBase;
+use Pimcore\Bundle\AdminBundle\Controller\Traits\DocumentTreeConfigTrait;
+use Pimcore\Event\AdminEvents;
+use Pimcore\Model\Document;
+use Pimcore\Model\Document\Page;
+use Pimcore\Routing\Dynamic\DocumentRouteHandler;
+use Symfony\Component\EventDispatcher\GenericEvent;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+
+
+/**
+ * @Route("/document")
+ *
+ * @internal
+ */
+class DocumentController extends ElementControllerBase
+{
+    use DocumentTreeConfigTrait;
+
+    final const DOCUMENT_ROOT_ID = 1;
```

Comment:
```suggestion
    final private const DOCUMENT_ROOT_ID = 1;
```

Reply:
I made it private only, final private is not possible, since private is not visible to others.